src/components/table-of-contents/index.jsx

```diff
@@ -5,35 +5,31 @@ import { useContext, useMemo, useRef } from 'preact/hooks';
 export const TocContext = createContext({ toc: null });
 
 export default function Toc() {
-	const ref = useRef(null);
 	const cache = useRef([]);
 
 	const { toc } = useContext(TocContext);
 
-	// eslint-disable-next-line
-	const items = useMemo(() => {
-		return toc !== null
+	const items = useMemo(() => (
+		toc !== null
 			? (cache.current = listToTree(toc))
-			: cache.current || [];
-	}, [toc]);
+			: cache.current || []
+	), [toc]);
 
-	if (items.length === 0) return <div ref={ref} />;
+	// TODO: Should we throw an error? No good reason to have a toc
+	// on the page if there is nothing to populate it with.
+	if (items.length === 0) return null;
 
 	return (
-		<div ref={ref}>
-			<nav onFocus={this.open}>
```

Comment:
This hasn't been hooked up for many years at this point, looks like we used to have a collapsible table of contents?

Reply:
yeah iirc we played around with that at some point. Not sure if we ever shipped it though.